docs/_ext/hidden_code_block.py

```diff
@@ -0,0 +1,127 @@
+"""Simple, inelegant Sphinx extension which adds a directive for a
```

Comment:
Why did we add this to our repo instead of as a dependency to the requirements? Also it doesn't credit the URL where we actually got it from..

Reply:
It is not uploaded to pypi.
The instructions for installation say to do it this way  -- http://scopatz.github.io/hiddencode/#installation

Reply:
We should at least mention the URL where we got it from.